### PR TITLE
[Fix] Space available calculation

### PR DESCRIPTION
**Pull request by BrettCleary**
Fixes space available calculation to download size + uncompressed size to match the new behavior in https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/pull/742

---

Use the following Checklist if you have changed something on the Backend or Frontend:

- [ ] Tested the feature and it's working on a current and clean install.
- [ ] Tested the main App features and they are still working on a current and clean install. (Login, Install, Play, Uninstall, Move games, etc.)
- [ ] Created / Updated Tests (If necessary)
- [ ] Created / Updated documentation (If necessary)

### CODE REVIEW

**Review by red-game-dev — APPROVED** [review]

File: src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx [hunk 1]
```diff
@@ -162,10 +162,12 @@ export default function DownloadDialog({
   const { i18n, t } = useTranslation('gamepage')
   const { t: tr } = useTranslation()
 
+  const compressedSize = gameInstallInfo?.manifest?.disk_size || 0
+  const gameDwnloadSize = gameInstallInfo?.manifest?.download_size || 0
```
Comment (awantoch) [inline src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx:166]:
```suggestion
  const gameDownloadSize = gameInstallInfo?.manifest?.download_size || 0
```

File: src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx [hunk 2]
```diff
@@ -162,10 +162,12 @@ export default function DownloadDialog({
...
   const uncompressedSize = useEstimatedUncompressedSize(
     platformToInstall,
-    gameInstallInfo?.manifest?.disk_size || 0,
-    gameInstallInfo?.manifest?.download_size || 0
+    compressedSize,
+    gameDwnloadSize
```
Comment (awantoch) [inline src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx:170]:
```suggestion
    gameDownloadSize
```

**Review by nyghtstalker — APPROVED** [review]
I can confirm the size difference in the PR branch vs the live release. The PR has a more accurate calculation.

image

File: src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx (hunk 1 above)
Comment (BrettCleary) [reply src/frontend/screens/Library/components/InstallModal/DownloadDialog/index.tsx]: fixed

**Pull request merged**